<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Intersection:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="output">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="AGroupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>AGroupBox</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>231</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Amin">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Amax">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Bmin">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Bmax">
          <property name="text">
           <string>200</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Cmin">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Cmax">
          <property name="text">
           <string>300</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="BGroupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>BGroupBox</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>40</y>
       <width>252</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>G</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="G3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="G1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="G2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="G4"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>F</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="F1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="F2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="F3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="F4"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>E</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="E1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="E2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="E3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="E4"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>351</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="intersection">
       <property name="text">
        <string>intersection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unions">
       <property name="text">
        <string>union</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SingleThread:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ElapseTimeShow">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Intersection:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="output2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MultipleThread:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ElapseTimeShow2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>470</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
